test_util/src/lib.rs

```diff
@@ -2273,14 +2273,21 @@ pub fn parse_strace_output(output: &str) -> HashMap<String, StraceOutput> {
   }
 
   let total_fields = total_line.split_whitespace().collect::<Vec<_>>();
+
+  let mut usecs_call_offset = 0;
   summary.insert(
     "total".to_string(),
     StraceOutput {
       percent_time: str::parse::<f64>(total_fields[0]).unwrap(),
       seconds: str::parse::<f64>(total_fields[1]).unwrap(),
-      usecs_per_call: None,
-      calls: str::parse::<u64>(total_fields[2]).unwrap(),
-      errors: str::parse::<u64>(total_fields[3]).unwrap(),
+      usecs_per_call: if total_fields.len() > 5 {
```

Comment:
**nitpick:** Could maybe use a comment about what / why there may be an extra field sometimes, yet other times not